### ISSUE TITLE
WCAG: Focus visible has disappeared from the edit buttons on the summary component  

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#909]
### Description of the bug

If you try to navigate a summary page using only a keyboard you will not be able to see what "endre" button has focus.
In this screenshot the focus is on one of the "endre" buttons
![Skjermbilde 2023-02-08 kl  15 40 14](https://user-images.githubusercontent.com/33222679/217561878-c876f038-7efa-4d84-9402-e7981d99e967.png)

Focus should be the same as for our other component. 

### Steps To Reproduce

Go to an app with a summary page
Try to navigate it with only a keyboard
All "endre" buttons should not have any focus

**Comment by bjosttveit** [comment]
## LGTM 🚀
